src/game_api/flags.hpp

```diff
@@ -292,30 +293,15 @@ const char* illumination_flags[]{
     "6: ",
     "7: Modulate brightness_multiplier",
     "8: ",
-    "9: ",
-    "10: ",
-    "11: ",
-    "12: ",
-    "13: ",
-    "14: ",
-    "15: ",
-    "16: ",
-    "17: Back layer",
-    "18: ",
-    "19: ",
-    "20: ",
-    "21: ",
-    "22: ",
-    "23: ",
-    "24: ",
-    "25: Enabled",
-    "26: ",
-    "27: ",
-    "28: ",
-    "29: ",
-    "30: ",
-    "31: ",
-    "32: ",
+    // type_flags 1-8:
+    "9: Follow camera",
+    "10: Follow entity",
+    "11: Static, Rectangle, Full brightness",
+    "12: unused",
+    "13: unused",
+    "14: unused",
+    "15: unused",
+    "16: unused",
```

Comment:
Everything past the 16th flag was wrong? Or was it moved to another flags field?

Reply:
those are bool's not flags, so i moved them to separate variable, also made sure that it's not breaking anything with struct and union stuff